pkg/cli/project_create.go

```diff
@@ -25,20 +25,24 @@ acorn project create acorn.io/username/new-project
 	})
 	// This will produce an error if the region flag doesn't exist or a completion function has already
 	// been registered for this flag. Not returning the error since neither of these is likely occur.
-	if err := cmd.RegisterFlagCompletionFunc("region", newCompletion(c.ClientFactory, regionsCompletion).complete); err != nil {
-		cmd.Printf("Error registering completion function for --region flag: %v\n", err)
+	if err := cmd.RegisterFlagCompletionFunc("default-region", newCompletion(c.ClientFactory, regionsCompletion).complete); err != nil {
+		cmd.Printf("Error registering completion function for --default-region flag: %v\n", err)
+	}
+	if err := cmd.RegisterFlagCompletionFunc("supported-region", newCompletion(c.ClientFactory, regionsCompletion).complete); err != nil {
+		cmd.Printf("Error registering completion function for --supported-region flag: %v\n", err)
 	}
 	return cmd
 }
 
 type ProjectCreate struct {
-	client ClientFactory
-	Region string `usage:"Region to create project in"`
+	client           ClientFactory
+	DefaultRegion    string   `usage:"Default region to create project in"`
```

Comment:
I think the usage string is a little unclear here. DefaultRegion is the region where new apps will be placed by default, right? "To create project in" doesn't really convey that imo.

Reply:
maybe `Default Region for project resources` or `Default region for created project`?